edb/schema/inheriting.py

```diff
@@ -955,7 +958,14 @@ def _propagate_field_alter(
         scls: so.InheritingObject,
         props: Tuple[str, ...],
     ) -> None:
-        for descendant in scls.ordered_descendants(schema):
+        descendant_names = [
+            d.get_name(schema) for d in scls.ordered_descendants(schema)
+        ]
+
+        for descendant_name in descendant_names:
+            descendant = schema.get(descendant_name, type=so.InheritingObject)
+            assert descendant, '.inherit_fields caused a drop of a descendant?'
```

Comment:
`get` will raise an error by itself, right?
If you want it deferred to this assert, pass a `default`